packages/angular/src/generators/utils/testing.ts

```diff
@@ -2,7 +2,7 @@ import type { Tree } from '@nx/devkit';
 import { names, readProjectConfiguration, updateJson } from '@nx/devkit';
 import { createTreeWithEmptyWorkspace } from '@nx/devkit/testing';
 import { Linter } from '@nx/eslint';
-import { UnitTestRunner } from '../../utils/test-runners';
+import { E2eTestRunner, UnitTestRunner } from '../../utils/test-runners';
```

Comment:
Not needed? The import was added, but there's no other change adding the usage.

Reply:
Ah yeah, missed leftover. Thanks!